src/styles/blocks/calendar.scss

```
@@ -0,0 +1,65 @@
+.calendar {
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+
+  &__list {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $gap-between-one-block;
+    width: ($one-column-width * 7) + 20px + $gap-between-one-block * 6;
+    padding: 10px;
+  }
+
+  $days:
+    mon 0,
+    tue 1,
+    wed 2,
+    thu 3,
+    fri 4,
+    sat 5,
+    sun 6;
+
+  @each $day, $value in $days {
+    &__list--start-day-#{$day}
+    &__days:first-child {
+      margin-left: $gap-between-one-block * $value
+        + $one-column-width * $value;
+    }
+  }
+
+  @for $month from 28 through 31 {
+    &__list--month-length--#{$month}
+    &__days:not(:nth-child(-n + #{$month})) {
+      display: none;
+    }
+  }
+
+  &__days {
+    height: $one-column-height;
+    width: $one-column-width;
+
+    text-align: center;
+    line-height: $one-column-height;
+
+    border: 1px solid black;
+    background-color: #eee;
+
+    transition: 0.5s;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.5s property1 ease, 0.5s property2 ease;
```